libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -186,6 +199,28 @@ public void addRoutes(@NonNull @Size(min = 1) List<? extends DirectionsRoute> di
     }
   }
 
+  public void addIdentifiableRoute(IdentifiableRoute directionsRoute) {
```

Comment:
Noticed that we're adding `addIdentifiableRoute` and `addIdentifiableRoutes` mimicking `drawRoute` and `drawRoutes` and was wondering what would happen if these are intermingled 🤔 Same question applies to `MapRouteLine`

Reply:
The only difference between the two is a property is added to the maps feature when an identifiable route is used. If the routes are drawn without identifiers they get the coloring defined in the xml.  This makes it a completely optional feature that doesn't impact anybody not using it. So they can be intermingled at runtime it's a question of whether or not the routes get the default coloring defined in the style or whether it's overridden.